pkg/core/data/transactions/fields.go

```diff
@@ -288,6 +288,11 @@ type SecretKey struct {
 	B *Scalar `json:"b"`
 }
 
+// IsEmpty is used when passing the SecretKey by value
+func (s SecretKey) IsEmpty() bool {
+	return s.A == nil && s.B == nil
+}
+
 // MSecretKey copy the USecretKey from rusk to transactions datastruct
```

Comment:
This comment is the same as the one for `USecretKey`, so it's a bit hard to know which one is going in which direction unless you look at the logic

Reply:
good point, will fix